draft-ietf-rtcweb-jsep.xml

```diff
@@ -607,7 +607,9 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
             used instead.</t>
 
             <t>When creating an IceCandidate object, JSEP
-            implementations MUST populate all of these fields.</t>
+            implementations MUST populate both the 
+            m=section index field and the MID field.</t>
+
```

Comment:
This doesn't seem quite right. Aren't you required to also populate ufrag and candidate